packages/nimble-components/CONTRIBUTING.md

```diff
@@ -445,10 +445,11 @@ When creating a new component, create a `*-matrix.stories.ts` Storybook file to
 
 Most user-visible strings displayed by Nimble components are provided by the client application and are expected to be localized by the application if necessary. However, some strings are built into Nimble components and are provided only in English. An application can provide localized versions of these strings by using design tokens set on label provider elements.
 
-There are currently 2 label providers:
+There are currently 3 label providers:
 
 -   `nimble-label-provider-core`: Used for labels for all components besides the table
 -   `nimble-label-provider-table`: Used for labels for the table (and table sub-components / column types)
+-   `nimble-label-provider-rich-text-editor`: Used for labels for the rich text editor
```

Comment:
```suggestion
The current label providers:

-   `nimble-label-provider-core`: Used for labels for all components without a dedicated label provider
-   `nimble-label-provider-table`: Used for labels for the table (and table sub-components / column types)
-   `nimble-label-provider-rich-text-editor`: Used for labels for the rich text editor
```

Reply:
The components README.md has similar language to update:

https://github.com/ni/nimble/blob/db5ebd42c334810bb29be9c86cee763da487b39b/packages/nimble-components/README.md#L103-L106

Reply:
Updated